docroot/themes/custom/twentyeighteen/sass/base/_base.scss

```diff
@@ -7,16 +7,18 @@ body {
 }
 
 body {
-  position: relative;
-  font-family: $body_font;
-  font-size: 1em;
   color: $dark-gray;
+  font-size: 1em;
+  font-family: 'Montserrat', Helvetica, Arial, sans-serif;
```

Comment:
Let's keep it as `$body_font.` $body_font variable has the same value as what you changed it to.